src/pages/CreateNewService.jsx

```diff
@@ -0,0 +1,8 @@
+import CreateNewServiceForm from '../components/Forms/CreateNewServiceForm';
+export default function CreateNewService() {
+	return (
+		<>
+			<CreateNewServiceForm />
+		</>
```

Comment:
As you are only returning one child you do not need the fragment.

```suggestion
			<CreateNewServiceForm />
```